registration/payments.py

```diff
@@ -10,12 +10,13 @@
 from .models import *
 
 client = Client(
+    timeout=2,
```

Comment:
2s seems rather quick, especially for a payments API where we're waiting for completion instead of just capture/auth first.  I'm thinking more in the range of 5 - 10s at least, though maybe 15s is a more appropriate lower bound.

It'd be good to try and instrument external API calls with Prometheus. Usually there's drop in libraries or wrappers for `requests` or similar client libraries but we could also just instrument a histogram with our payments code as well to get better visibility on these timings.